prawn-table.gemspec

```diff
@@ -20,6 +20,9 @@ Gem::Specification.new do |spec|
   spec.licenses = %w(PRAWN GPL-2.0 GPL-3.0)
 
   spec.add_dependency('prawn', '>= 1.3.0', '< 3.0.0')
+
+  spec.add_dependency('matrix') if RUBY_VERSION >= '3.1.0'
```

Comment:
Does prawn-table use matrix gem?

Reply:
This is what results from attempting to run specs on Ruby 3.1.0 when this line is omitted:

<img width="387" alt="Screen Shot 2022-02-07 at 9 50 13 AM" src="https://user-images.githubusercontent.com/421488/152843888-2f7a0684-a127-4122-b57f-c98ee6df993b.png">

So indirectly through the `require 'prawn'`.  If that's a development/spec only thing, we can either shift it to a development dependency or put it in the Gemfile.

Reply:
Oh, I see. Well, it's a transitive dependency issue. We have to bump prawn version dependency to a one that has `matrix` in its deps (a one that has not been released yet).

Reply:
Ok.  So what's the follow up action?

Reply:
We'll have to leave Ruby 3.1 out of the matrix until after Prawn release. Either extract that bit into a separate PR and merge the rest or hold this whole PR until then. Either works for me.